src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -2,18 +2,22 @@ import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { imagePaths } from '../constants';
+import orderBy from 'lodash/orderBy';
 
 const ProfileHeader = ({
-  userFullName: { first, middle, last },
+  userFullName: { first, middle, last, suffix },
   latestBranchOfService,
+  showBadgeImage,
 }) => {
-  const fullName = [first, middle, last].filter(name => !!name).join(' ');
+  const fullName = [first, middle, last, suffix]
+    .filter(name => !!name)
+    .join(' ');
 
   return (
     <div className="vads-u-background-color--gray-dark vads-u-color--white vads-u-margin-bottom--2 vads-u-padding-y--3 vads-u-display--flex vads-u-align-items--center vads-u-justify-content--center">
       <div className="medium-screen:vads-u-flex-direction--row usa-grid usa-grid-full vads-u-display--flex vads-u-flex-direction--column vads-u-align-items--center vads-u-width--full">
         <div className="medium-screen:vads-u-padding-left--8 usa-width-one-fourth">
-          {latestBranchOfService && (
+          {showBadgeImage && (
```

Comment:
just a heads up that I'm pretty sure we're gonna remove the badge from mobile (but keep it on tablet). Not sure if @tressaellen has decided on that yet.

Reply:
@erikphansen was this change recent, the mobile mocks have the badge included

Reply:
yeah just mentioned in a meeting I was in. fine to leave it like this for now! but it might change soon.